app/views/devise/sessions/new.html.erb

```diff
@@ -1,5 +1,11 @@
 <h2>Log in</h2>
 
+<% if current_user %>
+  <p> Hello!%></p>
+  <% else %>
+  <p>You are not signed in. Please sign in below.</p>
+<% end %>
+
```

Comment:
- [ ] After signing up I am not able to signin check the error I am getting
![role](https://user-images.githubusercontent.com/44745855/181905037-e7486fe3-e5f6-49ac-a7ef-f9d72e2512ea.png)

Reply:
Hello Reviewer 👋 
Please can you confirm that after signup, You verified the email by clicking the link in the Terminal:

<img width="1102" alt="image" src="https://user-images.githubusercontent.com/81635839/181905376-97b931bb-9f2f-4e3f-9c7d-571f849fc26e.png">

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/81635839/181905423-75afe7ce-6b56-4ea4-b2fc-404705e0f6aa.png">

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/81635839/181905459-4a1ff71b-db18-4073-8559-660547a1b90c.png">

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/81635839/181905477-b10d377e-a164-44c1-9c39-f84144059f17.png">


<img width="1009" alt="image" src="https://user-images.githubusercontent.com/81635839/181905397-4a430c18-83a4-4d7a-b875-89a6d7a1e1ae.png">

Thank you!